pandas/core/frame.py

```diff
@@ -2967,6 +2967,15 @@ def query(self, expr, inplace=False, **kwargs):
             The query string to evaluate.  You can refer to variables
             in the environment by prefixing them with an '@' character like
             ``@a + b``.
+
+            .. versionadded:: 0.25.0
```

Comment:
can you add an example in the  Examples section as well

Reply:
Done, but don't know what this means:

1 Warnings found:
	No extended summary found
Docstring for "pandas.DataFrame.query" correct. :)